docs/monitoring-usage.rst

```diff
@@ -0,0 +1,156 @@
+GCP projects and AWS accounts. It can also simply be the ID of
+a monitored project.
+
+Most often the authentication credentials will be determined
+implicitly from your environment. See :doc:`gcloud-auth` for
+more information.
+
+It is thus typical to create a client object as follows::
+
+    >>> from gcloud import monitoring
+    >>> client = monitoring.Client(project='target-project')
+
+If you are running in Google Compute Engine or Google App Engine,
+the current project is the default target project. This default
+can be further overridden with the :envvar:`GCLOUD_PROJECT`
+environment variable. Using the default target project is
+even easier::
+
+    >>> client = monitoring.Client()
+
+If necessary, you can pass in ``credentials`` and ``project`` explicitly::
+
+    >>> client = monitoring.Client(project='target-project', credentials=...)
+
+.. _Stackdriver account: https://cloud.google.com/monitoring/accounts/
+
+
+Monitored Resource Descriptors
+------------------------------
+
+The available monitored resource types are defined by *monitored resource
+descriptors*. You can fetch a list of these with the
+:meth:`~gcloud.monitoring.client.Client.list_resource_descriptors` method::
+
+    >>> for descriptor in client.list_resource_descriptors():
+    ...     print descriptor.type
+
+Each :class:`~gcloud.monitoring.resource.ResourceDescriptor`
+has a type, a display name, a description, and a list of
+:class:`~gcloud.monitoring.label.LabelDescriptor` instances.
+See the documentation about `Monitored Resources`_
+for more information.
+
+.. _Monitored Resources:
+    https://cloud.google.com/monitoring/api/v3/monitored-resources
+
+
+Metric Descriptors
+------------------
+
+The available metric types are defined by *metric descriptors*.
+They include `platform metrics`_, `agent metrics`_, and `custom metrics`_.
+You can list all of these with the
+:meth:`~gcloud.monitoring.client.Client.list_metric_descriptors` method::
+
+    >>> for descriptor in client.list_metric_descriptors():
+    ...     print descriptor.type
+
+See :class:`~gcloud.monitoring.metric.MetricDescriptor` and the
+`Metric Descriptors`_ API documentation for more information.
+
+.. _platform metrics: https://cloud.google.com/monitoring/api/metrics
+.. _agent metrics: https://cloud.google.com/monitoring/agent/
+.. _custom metrics: https://cloud.google.com/monitoring/custom-metrics/
+.. _Metric Descriptors:
+    https://cloud.google.com/monitoring/api/ref_v3/rest/v3/\
+    projects.metricDescriptors
+
+
+Time Series Queries
+-------------------
+
+A time series includes a collection of data points and a set of
+resource and metric label values.
+See :class:`~gcloud.monitoring.timeseries.TimeSeries` and the
+`Time Series`_ API documentation for more information.
+
+While you can obtain time series objects by iterating over a
+:class:`~gcloud.monitoring.query.Query` object, usually it is
+more useful to retrieve time series data in the form of a `pandas`_
```

Comment:
It might be useful to add `pandas` to our Inter-Sphinx mapping

```diff
diff --git a/docs/conf.py b/docs/conf.py
index 37842e9..6b2dcd8 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -292,5 +292,6 @@ html_context.update(
 intersphinx_mapping = {
     'httplib2': ('http://bitworking.org/projects/httplib2/doc/html/', None),
     'oauth2client': ('http://oauth2client.readthedocs.org/en/latest/', None),
+    'pandas': ('http://pandas.pydata.org/pandas-docs/stable/', None),
     'python': ('https://docs.python.org/', None),
 }
```

and then you can just refer to 

```
:mod:`pandas`
```